getting-started/vs-code-integration/introduction.md

```diff
@@ -10,60 +10,22 @@ position: 1
+* You can also use it to [add the Telerik Components to an existing project]({%slug getting-started-vs-code-integration-convert-project %}) with a few clicks.
 
-1. Press `Ctrl`+`Shift`+`P` in Windows/Linux or `Cmd`+`Shift`+`P` on Mac to open VSCode's extension launcher
 
-    ![launch Telerik Blazor VS Code extension](images/launch-extension.png)
 
-1. Type/Select `Telerik UI for Blazor Template Wizard: Launch` and press `Enter` to launch the extension
-
-1. The wizard will now guide you through the project creation steps:
-    1. choose a project name and location,
-    
-        ![choose Telerik project name and location](images/project-name-and-location.png)
-        
-    1. the type of project (WASM or Server-side) and your Telerik license (trial or commercial, depending on what you have),
-    
-        ![choose Blazor project type and Telerik license](images/project-type-and-license.png)
-    
-    1. add some sample pages to get you started (a grid with CRUD operations form a service, a form, charts),
-    
-        ![add sample pages](images/add-pages.png)
-    
-    1. choose a Theme.
-    
-        ![choose Telerik Theme](images/choose-theme.png)
-
-1. When you open the project, to run it:
-
-    1. Open the Terminal and navigate to the `Server` project folder. For a server project this is the folder with the project name, for a WASM project it will be the `Server` folder.
-
-    1. Execute `dotnet run` and navigate your browser to the link you see in the console output.
+## Get the Extension
 
-1. You now have a Blazor SPA app running!
+@[template](/_contentTemplates/common/general-info.md#vs-code-x-download)
 
+To access the VS Code extension press `Ctrl`+`Shift`+`P` in Windows/Linux or `Cmd`+`Shift`+`P` on Mac to open VSCode's extension launcher.
 
-## Troubleshooting
+![launch Telerik Blazor VS Code extension](images/launch-extension.png)
 
-If the extension does not work right (missing pages, does not start, etc.), you can try the following to clean up cache files and get it working again:
 
-1. Uninstall the extension from VS Code.
-2. Go to the extensions templates cache folder and delete the `BlazorT` and `KendoT` folders - they are where caches for the Telerik extensions are kept:
-    * on Windows, open `%localappdata%` 
-    * on Mac, open `/Users/[user_name]/.local/share/`
-3. [Install](https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard) the extension again.
+>note When using the Telerik Blazor Extension for Visual Studio Code to create new project or convert an existing one, an important point to take into consideration is that the extension is designed to get and apply the latest version of Telerik UI for Blazor that you have in your **local NuGet feed**. Therefore, it is up to you to control the version of Telerik UI for Blazor that will be used in your application (for best performance of the Telerik UI for Blazor components, we recommend using the latest stable version).
```

Comment:
Can you confirm this is the **local** nuget feed? AFAIK, the VS Code wizard puts `Version="*"` which means that the framework will get the latest version it can from any source. So, I suppose this should read something like "is designed to get and apply the latest version of Telerik UI for Blazor that you have access to. Therefore, you must ensure your system is set up to have the version you require. You can see how to use our online feed here or create a local feed here".

Reply:
Done.